geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -169,23 +182,45 @@ public ClusterManagementResult list(CacheElement filter) {
           "Cluster configuration service needs to be enabled");
     }
 
-    List<CacheElement> elements = new ArrayList<>();
+    List<RuntimeCacheElement> elements = new ArrayList<>();
```

Comment:
maybe there is a better variable name we could use here to convey that this is the thing we are building up to be returned at the end of the method?